### PR TITLE
Change the foreground quota limiter from experiemental to GAed.

### PR DESCRIPTION
**Pull request by BornChanger**
Signed-off-by: BornChanger <dawn_catcher@126.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Change the quota limiter for foreground request from experimental to GAed.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1684,33 +1684,35 @@ Raft Engine 相关的配置项。
...
 ### 前台限流
 
 用于前台限流相关的配置项。
 
 当 TiKV 部署的机型资源有限（如 4v CPU，16 G 内存）时，如果 TiKV 前台处理的读写请求量过大，以至于占用 TiKV 后台处理请求所需的 CPU 资源，最终影响 TiKV 性能的稳定性。此时，你可以使用前台限流相关的 quota 配置项以限制前台各类请求占用的 CPU 资源。触发该限制的请求会被强制等待一段时间以让出 CPU 资源。具体等待时间与新增请求量相关，最多不超过 [`max-delay-duration`](#max-delay-duration-从-v600-版本开始引入) 的值。
 
-> **警告：**
->
-> - 前台限流是 TiDB 在 v6.0.0 中引入的实验特性，不建议在生产环境中使用。
-> - 该功能仅适合在资源有限的环境中使用，以保证 TiKV 在该环境下可以长期稳定地运行。如果在资源丰富的机型环境中开启该功能，可能会导致读写请求量达到峰值时 TiKV 的性能下降的问题。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1695]:
```suggestion
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1684,33 +1684,35 @@ Raft Engine 相关的配置项。
...
 
 #### `foreground-cpu-time` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
 + 限制处理 TiKV 前台读写请求所使用的 CPU 资源使用量，这是一个软限制。
 + 默认值：0（即无限制）
 + 单位：millicpu （当该参数值为 `1500` 时，前端请求会消耗 1.5v CPU）。
++ 推荐设置：根据实验室测试结果，对 4 核以上的实例，使用默认值 0 即可；对 4 核实例，设置为 1000 到 1500 之间的值能取得比较均衡的效果；对 2 核实例，则不要超过 1200。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1701]:
```suggestion
+ 推荐设置：对于 4 核以上的实例，使用默认值 `0` 即可；对 4 核实例，设置为 `1000` 到 `1500` 之间的值能取得比较均衡的效果；对 2 核实例，则不要超过 `1200`。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1684,33 +1684,35 @@ Raft Engine 相关的配置项。
...
 
 #### `foreground-write-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
 + 限制前台事务写入的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：根据实验室测试结果，除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制的时候，一般不用使用默认值 0 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 50M。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1707]: 50M 是 50 兆吗？能否换算成可直接设置的值呢？
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 50M。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -1684,33 +1684,35 @@ Raft Engine 相关的配置项。
...
+
 
 #### `foreground-read-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入 </span>
 
 + 限制前台事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：根据实验室测试结果，除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制的时候，一般不用使用默认值 0 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 20M。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1714]:
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 20M。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
...
 + 限制处理 TiKV 前台读写请求所使用的 CPU 资源使用量，这是一个软限制。
 + 默认值：0（即无限制）
 + 单位：millicpu （当该参数值为 `1500` 时，前端请求会消耗 1.5v CPU）。
++ 推荐设置：对于 4 核以上的实例，使用默认值 `0` 即可；对 4 核实例，设置为 `1000` 到 `1500` 之间的值能取得比较均衡的效果；对 2 核实例，则不要超过 `1200`。
 
 #### `foreground-write-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
 + 限制前台事务写入的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 50M。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1706]:
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 50 M 以下。
```

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
...
+
 
 #### `foreground-read-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入 </span>
 
 + 限制前台事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置成 < 20M。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1713]:
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 20 MB 以内。
```

File: tikv-configuration-file.md [hunk 7]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
 
 + 单次读写请求被强制等待的最大时间。
 + 默认值：500ms
++ 推荐设置：根据实验室测试结果，一般使用默认值即可；如果实例出现了 OOM 或者是剧烈的性能都抖动，可以设置为 1S，使得请求被延迟调节的时间不超过 1 秒。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1687]:
```suggestion
+ 推荐设置：一般使用默认值即可。如果实例出现了内存溢出或者是剧烈的性能抖动，可以设置为 1 秒，使得请求被延迟调节的时间不超过 1 秒。
```

File: tikv-configuration-file.md [hunk 8]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
 
 + 单次读写请求被强制等待的最大时间。
 + 默认值：500ms
```
Comment (TomShawn) [inline tikv-configuration-file.md:1686]: 单位是否需要统一？这里是 ms，下面又是 s。其他几个配置项也是单位不统一。

File: tikv-configuration-file.md (hunk 7 above)
Comment (BornChanger) [reply tikv-configuration-file.md:1687]: 不要改 1S 是可以被识别的值.

File: tikv-configuration-file.md (hunk 8 above)
Comment (BornChanger) [reply tikv-configuration-file.md:1686]: 这个参数的值不是纯粹的数字, 而是可以读的值. 500ms, 1s, 等等

File: tikv-configuration-file.md [hunk 9]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
...
 + 限制处理 TiKV 前台读写请求所使用的 CPU 资源使用量，这是一个软限制。
 + 默认值：0（即无限制）
 + 单位：millicpu （当该参数值为 `1500` 时，前端请求会消耗 1.5v CPU）。
++ 推荐设置：对于 4 核以上的实例，使用默认值 `0` 即可；对 4 核实例，设置为 `1000` 到 `1500` 之间的值能取得比较均衡的效果；对 2 核实例，则不要超过 `1200`。
 
 #### `foreground-write-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
 + 限制前台事务写入的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 50 MB 以下。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1706]:
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 50M 以下。
```

File: tikv-configuration-file.md [hunk 10]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
...
+
 
 #### `foreground-read-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入 </span>
 
 + 限制前台事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 20 MB 以内。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1713]:
```suggestion
+ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 20M 以内。
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 11]
```diff
@@ -1684,33 +1684,34 @@ Raft Engine 相关的配置项。
...
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 50M 以下。
+
 
 #### `foreground-read-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入 </span>
 
 + 限制前台事务读取数据和 Coprocessor 读取数据的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对读带宽做限制，一般情况本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 20M 以内。
+
```
Comment (TomShawn) [inline tikv-configuration-file.md:1714]:
```suggestion
```

File: tikv-configuration-file.md [hunk 12]
```diff
@@ -1684,33 +1684,33 @@ Raft Engine 相关的配置项。
...
 + 默认值：0（即无限制）
 + 单位：millicpu （当该参数值为 `1500` 时，前端请求会消耗 1.5v CPU）。
++ 推荐设置：对于 4 核以上的实例，使用默认值 `0` 即可；对 4 核实例，设置为 `1000` 到 `1500` 之间的值能取得比较均衡的效果；对 2 核实例，则不要超过 `1200`。
 
 #### `foreground-write-bandwidth` <span class="version-mark">从 v6.0.0 版本开始引入</span>
 
 + 限制前台事务写入的带宽，这是一个软限制。
 + 默认值：0KB（即无限制）
++ 推荐设置：除非因为 `foreground-cpu-time` 设置不足以对写带宽做限制，一般情况下本配置项使用默认值 `0` 即可；否则，在 4 核及 4 核以下规格实例上，建议设置在 50M 以下。
+
```
Comment (TomShawn) [inline tikv-configuration-file.md:1708]:
```suggestion
```

**Pull request merged**